### ISSUE TITLE
Incorrect error message when data collection is empty

### ISSUE THREAD

**Issue by astrofrog** [glue-viz/glue-jupyter#122]
The following error message should be different when the data collection is empty:

```
    325         if data is None:
    326             if len(self._data) != 1:
--> 327                 raise ValueError('There is more than one data set in the data collection, please pass a data argument')
    328             else:
    329                 data = self._data[0]
```

**Comment by jsub1** [comment]
Resolved already by b0861f40

**Comment by dhomeier** [comment]
Closed by #171
